static/js/ReaderTextTableOfContents.jsx

```diff
@@ -810,8 +810,13 @@ class JaggedArrayNodeSection extends Component {
         if (this.contentCountIsEmpty(this.props.contentCounts[i])) { continue; }
         if (this.props.addressTypes[0] === "Talmud") {
           var enSection = Sefaria.hebrew.intToDaf(i);
-          var heSection = Sefaria.hebrew.encodeHebrewDaf(enSection);
-        } else {
+          var heSection = Sefaria.hebrew.encodeHebrewDaf(section);
```

Comment:
Let's see if we can do this using the data from the API.  It may not be practical, but I'd like to avoid special casing here.